Makefile

```diff
@@ -706,7 +706,7 @@ test-go-prepare: ensure-webassets bpf-bytecode rdpclient $(TEST_LOG_DIR) ensure-
 # Runs base unit tests
 .PHONY: test-go-unit
 test-go-unit: FLAGS ?= -race -shuffle on
-test-go-unit: SUBJECT ?= $(shell go list ./... | grep -v -e integration -e tool/tsh -e integrations/operator -e integrations/access -e integrations/lib )
+test-go-unit: SUBJECT ?= $(shell go list ./... | grep -v -e teleport/integration -e tool/tsh -e integrations/operator -e integrations/access -e integrations/lib)
```

Comment:
It feels a little odd to see the `teleport/` in this exclusion but not in the rest of the exclusions (and definitely had me confused for a half second) - I'm fairly sure this works but perhaps we ought to just add the `teleport/` to all of them ?

Reply:
Using regex should allow us to reduce the argument list :thinking: 
Let me suggest another approach